src/opera_disp_tms/generate_sw_disp_tile.py

```diff
@@ -111,7 +111,7 @@ def load_sw_disp_granule(granule: Granule, bbox: Iterable[int], frame: FrameMeta
 
     granule_xr = granule_xr.rio.clip_box(*bbox, crs='EPSG:3857')
     granule_xr = granule_xr.load()
-    valid_data_mask = granule_xr['connected_component_labels'] > 0
+    valid_data_mask = granule_xr['recommended_mask'] > 0
```

Comment:
For the two granules I looked at, `recommended_mask` data values were all zero (invalid data) and one (valid data), so `== 1` might read a little better here

Reply:
Done!